### PR TITLE
Fixed system user check for auto reassign

### PR DESCRIPTION
**Pull request by jpaidoussi**

### CODE REVIEW

File: casepro/cases/tests.py [hunk 1]
```diff
@@ -1338,6 +1338,18 @@ def test_reassign_case(self):
         case1.refresh_from_db()
         self.assertEqual(case1.assignee, self.moh)
 
+        # ensure that cases last reassigned by the system are reassigned again by the system
```
Comment (rudigiesler) [inline casepro/cases/tests.py]: Should this not be "are not reassigned again"?

**Comment by rudigiesler** [comment]
Just the one small comment from me, looks good otherwise

File: casepro/cases/tests.py (hunk 1 above)
Comment (jpaidoussi) [inline casepro/cases/tests.py]: Yes, thanks.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**